spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestTagDDL.java

```diff
@@ -45,19 +43,9 @@
 public class TestTagDDL extends ExtensionsTestBase {
   private static final String[] TIME_UNITS = {"DAYS", "HOURS", "MINUTES"};
 
-  @Parameters(name = "catalogName = {0}, implementation = {1}, config = {2}")
```

Comment:
@manuzhang what exactly is this PR "fixing"? Do we actually need to test this across all catalogs?

Reply:
You are correct since all existing catalogs load table with tag through SparkCatalog.